app/locales/en/translations.js

```diff
@@ -344,6 +344,25 @@ export default {
             my_projects: `My OSF Projects`,
             headline: `On the OSF`,
         },
+        'preprint-status-banner': {
+            message: {
+                base: `{{name}} uses {{reviewsWorkflow}}, therefore this preprint is`,
```

Comment:
The way it is right now, the second half of the content is displayed in multiple places and it's easier to keep that consistent if both places reference the same string. If there is a point where the language changes significantly then the strings and the templates would need to be updated but it would still be very simple.